ci/environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - pytest
   - pytest-cov
   - pre-commit
-  - julia
+  - julia=1.10.1
```

Comment:
Should be `1.10.0`, since `1.10.1` is what broke.

Reply:
Ah, sure!